one_fm/api/doc_methods/payroll_entry.py

```diff
@@ -30,6 +28,64 @@ def get_count_holidays_of_employee(self, employee):
 		holidays = days[0][0]
 	return holidays
 
+@frappe.whitelist()
+def fill_employee_details(self):
+	"""
+	This Function fetches the employee details and updates the 'Employee Details' child table.
+
+	Returns:
+		list of active employees based on selected criteria
+		and for which salary structure exists.
+	"""	
+	self.set('employees', [])
+	employees = self.get_emp_list()
+	
+	#Fetch Bank Details and update employee list 
+	get_bank_details(employees)

```

Comment:
Go with proper naming for this method as this method is setting the bank details under the employee details and not just getting. `set_bank_details` would be more appropriate.

Reply:
Okay. Sure.👍🏼